Why-Microsoft-supports-this-project.md

```diff
@@ -0,0 +1,3 @@
+# Why Microsoft supports this project?
+
+The project is highly supported and recognized by Microsoft due to its simplicity and innovative approach to enhance the UWP platform by utilizing helper functions, custom controls, and app services. It is also part of .NET Foundation.
```

Comment:
Maybe in the future its worth adding that the WCT also is a place  where contributors can bring and ideas to be discussed and potentially supported, where it would be more difficulty in other avenues. Not sure I am wording it right, but we can open an issue to fallow up later on this.